.github/workflows/_rust.yml

```diff
@@ -151,14 +151,15 @@ jobs:
         timeout-minutes: 5
         with:
           runtime: true
+        # These steps must be synchronized with build.sh and build.bat in `rust/gui-client`
```

Comment:
Should we just call the script instead?

Reply:
Maybe if we parameterized it. The script builds in release profile so it's way slower. The smoke tests run in debug profile